src/app/component/auth/components/restore-password/restore-password.component.ts

```diff
@@ -85,10 +85,11 @@ export class RestorePasswordComponent implements OnInit, OnDestroy {
     this.pageName.emit(page);
   }
 
-  sentEmail(userOwnSignIn: UserOwnSignIn): void {
+  sentEmail(): void {
+    const {email} = this.restorePasswordForm.value;
     this.loadingAnim = true;
     this.currentLanguage = this.localStorageService.getCurrentLanguage();
-    this.restorePasswordService.sendEmailForRestore(userOwnSignIn.email, this.currentLanguage)
+    this.restorePasswordService.sendEmailForRestore(email, this.currentLanguage)
```

Comment:
I am not sure but what does "public email: FormControl;" at row 26 for? As I see it is unused.

Reply:
You are right. I've deleted.